quality_control_stock_oca/models/qc_inspection.py

```diff
@@ -35,25 +32,6 @@ def _compute_picking(self):
             elif inspection.object_id._name == "stock.picking":
                 inspection.picking_id = inspection.object_id
 
-    @api.depends("object_id")
```

Comment:
Why removing this?

Reply:
There's no point in computing the lot field anymore if an inspection for each lot/serial number used in a stock picking is created, as this value will be assigned when creating the inspection.

Reply:
@pedrobaeza did you hace the opportunity to check Manuel's commets?

THX!

Reply:
But that shouldn't be the default, and we need to keep backwards compatibility.